src/services/apiMetadataService.js

```diff
@@ -701,6 +710,46 @@ const updateSubscriptionPolicy = async (req, res) => {
     }
 };
 
+const updateSubscriptionPolicies = async (req, res) => {
+    try {
+        if (config.generateDefaultSubPolicies) {
+            const msg = "Bulk updating of subscription policies is not allowed because 'generateDefaultSubPolicies' is enabled in the Developer Portal."
+            console.log(msg)
+            res.status(403).send(msg);
+        } else {
+            const { orgId } = req.params;
+            const subscriptionPolicies = req.body;
```

Comment:
Can we check the exact field in the body instead of just checking whether the body is precent